src/main/xml-parsers.js

```diff
@@ -319,7 +320,7 @@ export function parseListObjects(xml) {
   const parseCommonPrefixesEntity = responseEntity => {
     if(responseEntity){
       toArray(responseEntity).forEach((commonPrefix) => {
-        const prefix = toArray(commonPrefix.Prefix)[0]
+        const prefix = sanitizeObjectKey(toArray(commonPrefix.Prefix)[0])
         result.objects.push({prefix, size: 0})
```

Comment:
The same as previous comment:
```suggestion
        result.objects.push({prefix: sanitizeObjectKey(toArray(commonPrefix.Prefix)[0]),
                             size: 0})
```